src/main/scala/org/ergoplatform/settings/Algos.scala

```diff
@@ -1,20 +1,16 @@
 package org.ergoplatform.settings
 
 import io.iohk.iodb.ByteArrayWrapper
-import scorex.core.utils.ScorexEncoding
 import scorex.core.{VersionTag, versionToBytes}
 import scorex.crypto.authds.LeafData
 import scorex.crypto.authds.merkle.MerkleTree
-import scorex.crypto.hash.{Blake2b256, Digest32}
+import scorex.crypto.hash.Digest32
 import scorex.util._
 
-import scala.util.Try
+object Algos extends ErgoAlgos with scorex.core.utils.ScorexEncoding {
 
-object Algos extends ScorexEncoding {
-
-  type HF = Blake2b256.type
-
-  val hash: HF = Blake2b256
+  // due to ErgoAlgos from sigma extends scorex.util.ScorexEncoding where encoder is BytesEncoder
```

Comment:
please describe in more detail, why this override is here?

Reply:
Changed to:
```
  // ErgoAlgos in sigmastate extends scorex.util.ScorexEncoding where encoder is BytesEncoder
  // but here we use scorex.core.utils.ScorexEncoding where encoder is ScorexEncoder
```
After `ScorexEncoder` is moved (there is even a TODO for that) from `scorex.core` to `scorex.util` we can fix this ugliness.

Reply:
please add `After ScorexEncoder is moved (there is even a TODO for that) from scorex.core to scorex.util we can fix this ugliness.` in the code as well

Reply:
Added.